CRFModuleValidation/HeartRate_Measurement.json

```diff
@@ -9,16 +9,74 @@
     "steps":
     [
      {
-     "identifier"           : "cameraHeartRate",
-     "type"                 : "transform",
-     "sectionTransformer"   : { "resourceName": "HeartrateStep.json"}
+     "identifier"   : "name",
+     "type"         : "form",
+     "title"        : "Enter name",
+     "detail"       : "This is a string you can use to filter your results. For example, the first name of the person whose heart rate you are measuring.",
```

Comment:
2nd sentence isn't a complete sentence. Instead you could join the two clauses with an em-dash, like this:
"...filter your results. For example, the..." --> "...filter your results--for example, the..."